package.json

```diff
@@ -7,7 +7,9 @@
     "generate:api:clean": "rimraf webapp/src/app/core/modules/openapi",
     "generate:api:application-server-specs": "cd server/application-server && mvn verify -DskipTests=true && node ../../scripts/clean-openapi-specs.js",
     "generate:api:application-server-client": "npx openapi-generator-cli generate -i server/application-server/openapi.yaml -g typescript-angular -o webapp/src/app/core/modules/openapi --additional-properties fileNaming=kebab-case,withInterfaces=true --generate-alias-as-model",
-    "generate:api": "npm run generate:api:application-server-specs && npm run generate:api:clean && npm run generate:api:application-server-client"
+    "generate:api": "npm run generate:api:application-server-specs && npm run generate:api:clean && npm run generate:api:application-server-client",
+    "generate:api:intelligence-service-specs": "python -m server.intelligence-service.app.generate_openapi_yaml",
+    "generate:api:intelligence-service-client": "npx @openapitools/openapi-generator-cli generate -i server/intelligence-service/openapi.yaml -g java -o server/intelligence-service/java-client"
```

Comment:
This needs some spring boot specific config:
https://openapi-generator.tech/docs/generators/java

As far as I can see:
library=resttemplate
Then since spring-web 6.1.11 -> useJakartaEe=true
performBeanValidation=true
generateClientAsBean=true

There are also other things missing such as:
- apiPackage
- artifactDescription
- groupId
- invokerPackage
- modelPackage